src/parser/paladin/protection/modules/Abilities.js

```diff
@@ -242,6 +242,31 @@ class Abilities extends CoreAbilities {
         },
         enabled: combatant.hasTalent(SPELLS.REPENTANCE_TALENT.id),
       },
+      {
+        spell: SPELLS.DIVINE_TOLL,
```

Comment:
All you need to do to add this is add `DivineToll.ts` to the CombatLogParser

Reply:
Good catch, I have corrected it to use DivineToll.ts.
Thanks,